plugins/gfm/src/main/kotlin/GfmPlugin.kt

```diff
@@ -251,33 +255,36 @@ open class CommonmarkRenderer(
         distinct.values.forEach { entry ->
             val (instance, sourceSets) = entry.getInstanceAndSourceSets()
 
-            append(sourceSets.joinToString(prefix = "#### [", postfix = "]") { "${it.moduleDisplayName}/${it.displayName}" })
+            append(sourceSets.joinToString(prefix = "[", postfix = "]") { it.displayName })
```

Comment:
it would be super nice to extract label generation to a fun. Right now it is in 3 places.

Reply:
Yeah, you are right